### PR TITLE
🐛 Bug #2440 Submitted Data Modal Filters

### PR DESCRIPTION
**Pull request by demariadaniel**
# Description of changes

* Updates Submitted Data Filter Modal to correctly set URL Param Donor ID State & accurately Query + Count results to match user input
* Filters accurately using both DonorIds and SubmitterDonorIds
* Testable using program HNAHAL-CA and entering '250441, 250442, DASH-2' into Modal textbox

## Type of Change

- [x] Bug
- [ ] Dependency updates
- [ ] Feature
- [x] Refactoring
- [ ] Release candidate
- [ ] Styling
- [x] Testing
- [ ] Other (please describe)

## Checklist before requesting review

- Design (select one):
  - [ ] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [x] No changes to UI
- Back-end (select one):
  - [ ] Back-end PRs merged, tested on develop, and linked in this PR
  - [x] No back-end changes
- [x] Manually tested changes
- [ ] Added copyrights to new files
- [ ] Connected ticket to PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "argo-platform-ui",
-  "version": "1.117.0",
+  "version": "1.118.0",
```
Comment (demariadaniel) [inline package.json:3]: Sorry I based this off the RC!

File: components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx [hunk 2]
```diff
@@ -61,13 +61,15 @@ export default function FilterModal({
   //make matchID dynamic using useState
   const [numMatched, setNumMatched] = useState(0);
   const [numUnmatched, setNumUnmatched] = useState(0);
-  const [matchedIds, setMatchedIds] = useState([]);
+  const [matchedIds, setMatchedIds] = useState('');
```
Comment (ciaranschutte) [inline components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx:64]: https://github.com/icgc-argo/platform-ui/blob/f8b805df430da257b00dc4acc0e11202ad519bcb/components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx#L116

This state is still being set as an array. Sometimes empty string sometimes array

**Review by ciaranschutte — APPROVED** [review]
Looks good.

probably not a hotfix if we're merging into develop

**Comment by daniel-cy-lu** [comment]
Manual Tested and the hotfix worked beautifully to get rid of the two bugs Dan and I were working on. 1. incorrect "Matched/Unmatched" counts 2. Result table doesn't work on multiple entry. 

I do still see the POST error we saw yesterday, but it's not affecting our features. 

Screenshot of the error:
 
Screenshot 2022-12-13 at 3 22 58 PM

File: package.json (hunk 1 above)
Comment (samrichca) [reply package.json:3]: can you update your PR with 1.117.0 in package.json & package-lock.json?

**Comment by demariadaniel** [comment]
> Looks good.
> 
> probably not a hotfix if we're merging into develop

Thanks, updated PR title and connected PR to ticket

**Pull request merged**